src/middlewares/authorization-verifier.js

```diff
@@ -0,0 +1,33 @@
+const errors = require('@feathersjs/errors');
+const jwt = require('jsonwebtoken');
+const config = require('config');
+
+module.exports = function createAuthorizationVerifier(userRepository) {
+  return {
+    validateUserToken: extractAuthorizationToken,
+    validateUserTokenDb: verifyUserId,
```

Comment:
It seems verify User is a good name for the exported function why change it to `validateUserTokenDb`